docs/ManualClusterSetup.md

```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
```

Comment:
I've never liked how we say 2 servers here. We're talking about manual cluster setup, why do we require any particular number?